Adding in ESLint and fixing basic errors to match #104

jonathanKingston opened the pull request.

MoOx commented:
Very nice !

Can you also:
- use a symlink for eslintignore (-> .gitignore)
- remove all trace of jshint and jscs (.rc + package.json refs)

That would be very cool.

jonathanKingston commented:
Done.

- Made the checks run the .gitignore like the jshint did before.
- Removed the babel-eslint lines as just noise.
- Removed mentions of jscs and jshint

MoOx commented:
Awesome ! Thanks a lot.
